examples/nextjs-cldogimage/components/Header.jsx

```diff
@@ -0,0 +1,44 @@
+export default function Header() {
+  return (
+    <header className="mb-12 mt-0 w-full bg-blue-950 py-4 text-center text-white">
+      <div className="mx-auto flex max-w-screen-lg flex-col items-center justify-between px-4 sm:flex-row">
+        <h1 className="flex items-center gap-3 text-xl font-semibold">
+          <a
+            href="https://next.cloudinary.dev"
+            className="text-blue-200 underline-offset-2 hover:underline"
+            target="_blank"
+          >
+            Next Cloudinary
+          </a>
+          <code className="font-mono text-lg">{`<CldOgImage />`}</code>
```

Comment:
i realize for simplicity, having this same value is ideal, but i think it can be a big confusing in the context of the App Router that doesn't use this, even with your note

what if the <Header> component took a prop for the component/function being used? then you can dynamically pass it in from the layout. bonus, make everything dynamic in here prop-based so that you only need to configure those things in the layout from now on

Reply:
Makes sense yeah. I can change it to the function name that we use in the app router instead of this component name.

At first I kept it as it is for simplicity but yea it definitely gets confusing